.whatsnew.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://raw.githubusercontent.com/dotnet/docs-tools/main/WhatsNew.Infrastructure/Configuration/reposettings.schema.json",
+    "docSetProductName": ".NET Aspire",
+    "rootDirectory": "docs/",
+    "docLinkSettings": {
+        "linkFormat": "relative",
+        "relativeLinkPrefix": "../"
+    },
+    "navigationOptions": {
+        "maximumNumberOfArticles": 3,
+        "tocParentNode": "Latest documentation updates",
+        "repoTocFolder": "docs/whats-new",
+        "indexParentNode": "Latest documentation updates",
+        "repoIndexFolder": "docs/whats-new"
+    },
+    "areas": [
```

Comment:
The "areas" will be displayed in the final article in the order they are listed here. You may want to move some up or down. (I made a few suggestions below.